modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -44,49 +44,52 @@ def parse_table_row(row)
       if row.css('a').try(:first) && (url = row.css('a').first['href'])
         return if url.starts_with?('#') || url == 'help.asp'
 
-        begin
-          parse_form_row(row, url)
-        rescue
```

Comment:
@bastosmichael  removing the rescue here means that if an error is thrown on any form the reloaded dies. The purpose of the rescue was to log the bad form but still be able to move on to the next form. There are errors in the current urls for example some forms can only be accessed from the VA network so trying to open the URL throws an error.

Reply:
I was able to get it to successfully run all the way through without a single error which is why I removed it but what you're saying makes sense so I'll bring it back.

Reply:
yea, I mean if we could trust the dataset I would be less inclined to rescue, but as we know its a bit of a mess

Reply:
I've tested this again and again and the data set is good now and every url is a proper pdf with a few of the minor tweaks to how we segment dates etc but good to be cautious here for sure.